packages/discovery-provider/integration_tests/tasks/user_bank_mock_transactions.py

```diff
@@ -245,6 +246,179 @@
     )
 )
 
+# Purchase of album for $1 USDC
+mock_valid_album_purchase_tx = GetTransactionResp.from_json(
+    json.dumps(
+        {
+            "jsonrpc": "2.0",
+            "result": {
+                "slot": 227246439,
+                "transaction": {
+                    "signatures": [MOCK_SIGNATURE],
+                    "message": {
+                        "header": {
+                            "numRequiredSignatures": 1,
+                            "numReadonlySignedAccounts": 0,
+                            "numReadonlyUnsignedAccounts": 8,
+                        },
+                        "accountKeys": [
+                            FEE_PAYER,
+                            SENDER_USDC_USER_BANK_ADDRESS,
```

Comment:
I'm curious which set of mock transactions were easier to extend? These were written later and I thought it was helpful to use constants for the addresses to make this easier to read. But I'm unsure if that's actually true 😅

Reply:
tbh we just copy/pasted so they were the same lol